source/whats-new.txt

```diff
@@ -12,13 +12,34 @@ What's New
 
 Learn what's new in:
 
+* :ref:`Version 4.6 <version-4.6>`
 * :ref:`Version 4.5 <version-4.5>`
 * :ref:`Version 4.4 <version-4.4>`
 * :ref:`Version 4.3 <version-4.3>`
 * :ref:`Version 4.2 <version-4.2>`
 * :ref:`Version 4.1 <version-4.1>`
 * :ref:`Version 4.0 <version-4.0>`
 
+
+.. _version-4.6:
+
+What's New in 4.6
+-----------------
+
+New features of the 4.6 Java driver release include:
+
+- The buffer pool is now shared across all ``MongoClient`` instances. By
+  default, the buffer pool prunes buffers that have been idle for one
+  minute.
+- Added an API to specify AWS credentials for Client-Side Field Level
+  Encryption (CSFLE) the same way as specifying authentication credentials
+  using :ref:`MONGODB-AWS <mongodb-aws-auth-mechanism>`.
+- Added CSFLE library support for Apple M1 ARM-based hardware.
+- Added the ``DnsClient`` and ``DnsClientProvider`` interfaces to allow
+  custom DNS client implementations. Added a service provider interface (SPI)
+  to specify the custom DNS client using ``java.util.ServiceLoader``. The
+  driver uses a default implementation if the application doesn't specify one.
```

Comment:
Saying _"Added a service provider interface (SPI)"_ after _"Added the ``DnsClient`` and ``DnsClientProvider`` interfaces"_ is weird because `DnsClient` and `DnsClientProvider` together is the SPI we added. I propose to write the following:

> Added the ``DnsClient`` and ``DnsClientProvider`` interfaces which represent a service provider interface (SPI) that allows customizing the DNS client used by the driver.

We don't even need to mention ``java.util.ServiceLoader``, because the documentation of the `DnsClientProvider` states _"The driver discovers implementations of this interface via `java.util.ServiceLoader`."_

Reply:
I don't mind repeating in the reference docs what's already in the API docs.  We do that in many places I am sure.